pippy/PipelineSchedule.py

```diff
@@ -645,155 +711,112 @@ def step(self, microbatches):
         Rank 1:    1F 1F 1F 1F 3F3B 3F 3F 3F
         """
 
+        n_microbatch = len(microbatches)
+        assert (
+            n_microbatch % self.world_size == 0
+        ), f"Looped DFS schedule requires microbatch_size ({n_microbatch}) to be a multiple of the number of \
+pipelined ranks ({self.world_size})"
```

Comment:
aha, shall we rename `world_size` as `pp_group_size`?

Reply:
Updated, thanks!